### ISSUE TITLE
Specification of Hyperparameters

### ISSUE THREAD

**Issue by heindorf** [dice-group/Ontolearn#29]
Currently, there is no explicit mechanism for managing hyperparameters in OntoPy.

Scikit-learn uses the methods `get_params` and `set_param` to manage hyperparameters (defined in `sklearn.base.BaseEstimator`).

The advantage of reusing this base class would be that scikit-learn's mechanisms for hyperparameter optimization can be reused for OntoPy (e.g., `GridSearch`).
